AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginIntegrationTests/DataStoreLocalUsageTests.swift

```diff
@@ -0,0 +1,289 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+
+import AmplifyPlugins
+import AWSMobileClient
+import AWSPluginsCore
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSDataStoreCategoryPlugin
+
+class DataStoreLocalUsageTests: LocalUsageIntegrationTestBase {
+
+    func testOneSort() throws {
+        _ = saveData(num: 20)
+
+        let queryFinished = expectation(description: "Query post completed")
+        var posts = [Post]()
+
+        Amplify.DataStore.query(Post.self,
+                                sort: .ascending(Post.keys.rating)) { res in
+            switch res {
+            case .success(let returnPosts):
+                posts = returnPosts
+                queryFinished.fulfill()
+            case .failure(let error):
+                XCTFail("Error quering posts, \(error)")
+            }
+        }
+        wait(for: [queryFinished], timeout: 10)
+
+        var previousRating: Double = 0
+        for post in posts {
```

Comment:
Since we are clearing the DB each time, ideally the tests are consistently get 20 posts because it was set up that way. assert post count == 20

Reply:
Added